internal/webserver/webserver.go

```diff
@@ -145,8 +154,10 @@ func (n kubeFilter) handleRequest(request *http.Request, username string, select
 	log.V(4).Info("updating RawQuery", "query", q.Encode())
 	request.URL.RawQuery = q.Encode()
 
-	log.V(4).Info("Updating the token", "token", n.bearerToken)
-	request.Header.Set("Authorization", fmt.Sprintf("Bearer %s", n.bearerToken))
+	if len(n.bearerToken) > 0 {
```

Comment:
To avoid interferences with the empty string due to run on out-of-cluster mode, we're skipping the update of the token.

Running in out-of-cluster mode, the transport will inherit the cluster administrator certificates, so we're good from the authorization perspective.